velox/docs/functions/spark/datetime.rst

```diff
@@ -66,10 +66,17 @@ These functions support TIMESTAMP and DATE input types.
         SELECT dayofweek('2009-07-30'); -- 5
         SELECT dayofweek('2023-08-22 11:23:00.100'); -- 3
 
-.. function:: dow(x) -> integer
+.. spark::function:: dow(x) -> integer
 
     This is an alias for :func:`day_of_week`.
 
+.. spark::function::from_unixtime(unixTime, format) -> string
+
+    Converts ``unixTime`` (elapsed seconds since UNIX epoch) to a date time string with
+    specified ``format``. The result is adjusted according to local time zone. ::
+
+        SELECT from_unixtime(100, "yyyy-MM-dd HH:mm:ss"); -- "1970-01-01 00:01:40"
```

Comment:
Can we add more examples of different formats?

Reply:
Added more examples. If invalid format is used, runtime exception will be thrown, consistent with Spark.

Reply:
> If invalid format is used, runtime exception will be thrown.

Can we document above? And add test to ensure exception is thrown.